front/app/containers/EventsShowPage/components/Container.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+// styling
+import {
+  contentFadeInDelay,
+  contentFadeInDuration,
+  contentFadeInEasing,
+  pageContentMaxWidth,
+} from '../styleConstants';
+import styled from 'styled-components';
```

Comment:
This component is based on the idea page container, and controls animations + structural styling.